packages/client/src/features/viewport/viewport-handler.ts

```diff
@@ -134,25 +133,29 @@ export class MoveViewportHandler implements IActionHandler {
  * Handles zooming in and out of the viewport.
  */
 @injectable()
-export class ZoomViewportHandler implements IActionHandler {
+export class ZoomHandler implements IActionHandler {
     @inject(EditorContextService)
     protected readonly editorContextService: EditorContextService;
     @inject(TYPES.IActionDispatcher)
     protected readonly actionDispatcher: IActionDispatcher;
 
     handle(action: Action): Action | void {
-        if (ZoomViewportAction.is(action)) {
-            return this.handleZoomViewport(action);
+        if (ZoomAction.is(action)) {
```

Comment:
Nit: Unncessary typecheck. If we have only one action that is handled you can define the correct type in the method signature directly

```ts
handle(action: ZoomViewportAction)
````